compose/postgres.yml

```diff
@@ -4,9 +4,9 @@ services:
     image: postgres
     restart: always
     environment:
-      POSTGRES_DB: betting
+      POSTGRES_DB: internship
```

Comment:
please revert to previous values and add this to a separate change list so you don't commit local setup in the future

Reply:
do the same thing for password